src/graph.cpp

```diff
@@ -33,20 +34,33 @@ Graph::Graph(node_id_t num_nodes, int num_inserters): num_nodes(num_nodes) {
     supernodes[i] = Supernode::makeSupernode(num_nodes,seed);
     parent[i] = i;
   }
-  num_updates = 0; // REMOVE this later
```

Comment:
Why is this no longer temporary?

Reply:
I added it a long time ago and thought it would just be a quick variable for testing but its actually really useful to have so we can just keep it.